### PR TITLE
Feature/rw customize notification

### LINKED ISSUE #75: Bug: Notifications with incompatible browsers
### Feature Categories:
- React

**Describe the feature/Bug your gonna work on**
- After implementing our `Notifications` functionality, we realized that this [API isn't compatible](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API) with all browsers. The biggest gap in this functionally seems to be Safari on iOS (though Chrome on iOS isn't working for me either).

**Additional context**
- [This article](https://stackoverflow.com/questions/60827563/workaround-for-missing-web-push-on-safari-for-my-pwa) might have some solutions!

### LINKED ISSUE #79: Customize notification
### Feature Categories:

- React; 
- Service Worker/Notification API

**Describe the feature/Bug your gonna work on**
- Customize the notification that gets pushed when timer ends, in displayNotification function
- Customize body, icon, vibration pattern, actions

### PR DESCRIPTION
**Pull request by rwilliams659**
# Som Timer Pull Request Form

## Description
- Toast notifications were implemented with the package react-toastify in order to still be able to show notifications if the Notifications API is not available on the users' browser or if they do not have permissions set to allow the notifications. 
- Notification API content slightly adjusted. I cannot see any icons on my notifications, but they may work for others. This was hard to test/tell so I did not spend a ton of time on it. 

Here's a preview of the toast notification (only available in the specific browser tab/mobile app view):
Som Timer
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [ ] No
- [X] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`Your message here`
# Message for reviewer:
`Your message here`
# Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

resolve #75 
resolve #79

### CODE REVIEW

File: src/helpers/notificationHelpers.js [hunk 1]
```diff
@@ -13,18 +13,18 @@ export const displayNotification = () => {
   if (Notification.permission === 'granted') {
     navigator.serviceWorker.getRegistration().then(reg => {
       const options = {
-        body: 'Return to Som Timer to choose your break content',
+        body: 'Choose your break content on Som Timer',
         icon: '/favicon.ico',
-        vibrate: [100, 50, 100],
+        vibrate: [300, 300, 300],
```
Comment (jkwest-93) [inline src/helpers/notificationHelpers.js:18]: Thanks for updating this pattern!!

**Review by jkwest-93 — APPROVED** [review]
Great find on this type of notification! I think this is a phenomenal way to still engage users who can't connect to the Notifications API.

**Review by jkwest-93 — CHANGES_REQUESTED** [review]
I'm good for merge once the Travis stuff is passing!

**Review by jkwest-93 — APPROVED** [review]
Great work fixing the weird `CountdownTimer` stuff! Glad we got these tests sorted out.

File: src/Components/App/App.js [hunk 2]
```diff
@@ -8,12 +8,19 @@ import Stats from '../Stats/Stats'
...
     <section className="App">
+      <ToastContainer 
+        position="top-center"
+        autoClose={10000}
+        closeOnClick
+        pauseOnHover={false}
+      />
       <Header />
       <Switch>
         <Route exact path="/" render={() => <HomeContainer />} />
```
Comment (Abdeboskey) [inline src/Components/App/App.js:26]: So toasty!! I love how simple this is!

**Review by Abdeboskey — APPROVED** [review]
This is a really great way to do notifications!! I love the style and I feel like I'll use this a lot in the future.

**Pull request merged**